src/input/path_base.hh

```diff
@@ -142,14 +142,14 @@ public:
     std::string as_string() const;
 
     /**
-     * @brief Gets name of descendant Record of Abstract.
+     * @brief Gets type and value of record tag
```

Comment:
Not clear that type and value are the same thing.

Gets value of the record tag, which determines its type.